index.css

```diff
@@ -20,30 +46,40 @@ p{
     font-size: 30px;
     margin-left: 20px;
 }
+/*End Brand block  */
 
-.page .header{
-    width: 100%;
-    margin-bottom: 5px;
-    border: 1px solid red;
-    padding-left: 60px;
+/* Start Navigation block*/
+.navigation{
+    width: 33%;
```

Comment:
from the html, this class was intend to be used as container for main menu block. So it should be menu-container as stand alone component